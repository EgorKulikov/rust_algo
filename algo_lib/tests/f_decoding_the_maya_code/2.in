6 4
AA
AB
BC
CD
AA
AA
1 2
2 3
3 4
4 5
5 6
1 AAABCD
2 AABCD
3 BCD
1 DCBA
