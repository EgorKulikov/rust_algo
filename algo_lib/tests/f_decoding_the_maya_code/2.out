9
5
2
0
